shared-bindings/bitmaptools/__init__.c

```diff
@@ -294,13 +319,16 @@ MP_DEFINE_CONST_FUN_OBJ_KW(bitmaptools_rotozoom_obj, 0, bitmaptools_obj_rotozoom
 //|     :param float factor1: The proportion of bitmap 1 to mix in
 //|     :param float factor2: The proportion of bitmap 2 to mix in.  If specified as `None`, ``1-factor1`` is used.  Usually the proportions should sum to 1.
 //|     :param displayio.Colorspace colorspace: The colorspace of the bitmaps. They must all have the same colorspace.  Only the following colorspaces are permitted:  ``L8``, ``RGB565``, ``RGB565_SWAPPED``, ``BGR565`` and ``BGR565_SWAPPED``.
+//|     :param bitmaptools.BlendMode blendmode: The blend mode to use. Default is Normal.
+//|     :param int skip_source1_index: bitmap palette index in source_bitmap_1 that will not be blended, set to None to blend all pixels
```

Comment:
I'm not sure saying "palette index" is correct, because this code operates on RGB and L (luminance) images, not palette images.

Reply:
Ah, good point. I'll word it better.